Libraries/acdriver/Sources/Options.cpp

```diff
@@ -19,6 +19,7 @@ Options() :
     _errors(false),
     _notices(false),
     _compressPNGs(false),
+    _targetDevice({}),
```

Comment:
This isn't necessary — `std::vector` has a default constructor that makes an empty one.